tests/dualtor/test_ipinip.py

```diff
@@ -137,5 +148,5 @@ def test_decap_standby_tor(
     logging.info("send encapsulated packet from ptf t1 interface %s", ptf_t1_intf)
     with tunnel_traffic_monitor(tor, existing=False):
         testutils.send(ptfadapter, int(ptf_t1_intf.strip("eth")), encapsulated_packet, count=10)
-
-    testutils.verify_no_packet_any(ptfadapter, exp_pkt, ports=[exp_ptf_port_index])
+        time.sleep(2)
+        verify_downstream_packet_to_server(ptfadapter, exp_ptf_port_index, exp_pkt)
```

Comment:
Instead of creating this custom method could we call `ptfadapter.dataplane.flush()` after verification to make sure subsequent verifications don't have access to these packets?